gradle/libs.versions.toml

```diff
@@ -42,12 +42,13 @@ logback = "1.4.11"
 maven = "3.9.4"
 mavenResolver = "1.9.15"
 mockk = "1.13.7"
-okhttp = "4.11.0"
+okhttp = "5.0.0-alpha.11"
 postgres = "42.6.0"
 postgresEmbedded = "1.0.1"
 reflections = "0.10.2"
 retrofit = "2.9.0"
 retrofitConverterKotlinxSerialization = "1.0.0"
+s3 = "0.29.0-beta"
```

Comment:
Also here, please use the latest stable / non-beta version, if possible.

Reply:
[0.32.4-beta](https://mvnrepository.com/artifact/aws.sdk.kotlin/s3/0.32.4-beta) is the latest version of the official Kotlin SDK provided by Amazon. We could try to use their Java SDK, [2.20.162](https://mvnrepository.com/artifact/software.amazon.awssdk/s3/2.20.162) But I believe it will defy the purpose of having the best version available from the vendor, which uses actual Kotlin features.

Reply:
Ok, I see, the AWS SDK for Kotlin is a developer preview, and there hasn't been any non-beta release since version [0.16.0](https://mvnrepository.com/artifact/aws.sdk.kotlin/s3/0.16.0) from 2022. So let's stick to the latest beta version then.

Reply:
Using the latest now. Although not important now that we are refactoring to use the java library